tests/data/test_dataset_splits.py

```diff
@@ -0,0 +1,242 @@
+import tempfile
+from pathlib import Path
+from typing import Any
+
+import pandas as pd
+import pytest
+from pandas import Timestamp
+
+from paint import PAINT_ROOT
+from paint.data.dataset_splits import DatasetSplitter
+
+
+@pytest.mark.parametrize(
+    "remove_unused_data, split_type, training_size, validation_size, expected_splits",
```

Comment:
Can we add one wrong string for `split_type` here to make sure the else block with the exception is also covered by the tests?